doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -242,6 +242,9 @@ nRF Desktop
 
 * Updated:
 
+  * The nRF desktop configurations that enable :ref:`nrf_desktop_bluetooth_guide_fast_pair` use MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) instead of ``B0`` bootloader.
+    This is done to support firmware updates using both :ref:`nrf_desktop_dfu` and :ref:`nrf_desktop_dfu_mcumgr`.
```

Comment:
```suggestion
  * The nRF desktop configurations that enable :ref:`nrf_desktop_bluetooth_guide_fast_pair`.
     The configurations use the MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) instead of the ``B0`` bootloader.
    This is done to support firmware updates using both :ref:`nrf_desktop_dfu` and :ref:`nrf_desktop_dfu_mcumgr` modules.
```